a3p-integration/proposals/a:vaults-auctions/eval.sh

```diff
@@ -0,0 +1,9 @@
+# we have an eval.sh so we can run prepare.sh before the rest
+
+echo "[$PROPOSAL] Running prepare.sh"
+./prepare.sh
```

Comment:
`prepare` is for chain-halting upgrade proposals. There should be no `prepare.sh` in a CoreEvalProposal.

Please remove the indirection:
```suggestion
./saveAuctionInstance.js
```